### PR TITLE
Implement lint task in workspaces instead of at root

### PR DESCRIPTION
**Pull request by mehulkar**
Before:
```
digraph {
	compound = "true"
	newrank = "true"
	subgraph "root" {
		"[root] //#lint" -> "[root] ___ROOT___"
	}
}
```

After:
```
digraph {
	compound = "true"
	newrank = "true"
	subgraph "root" {
		"[root] @turbo/codemod#lint" -> "[root] ___ROOT___"
		"[root] @turbo/pack-test-harness#lint" -> "[root] ___ROOT___"
		"[root] @vercel/experimental-nft#lint" -> "[root] ___ROOT___"
		"[root] @vercel/experimental-nft-next-plugin#lint" -> "[root] ___ROOT___"
		"[root] @vercel/turbo-tracing-test-app#lint" -> "[root] ___ROOT___"
		"[root] @vercel/turbopack-next#lint" -> "[root] ___ROOT___"
		"[root] @vercel/turbopack-node#lint" -> "[root] ___ROOT___"
		"[root] @vercel/turbopack-runtime#lint" -> "[root] ___ROOT___"
		"[root] @vercel/webpack-nft#lint" -> "[root] ___ROOT___"
		"[root] benchmark#lint" -> "[root] ___ROOT___"
		"[root] cargo-sweep-action#lint" -> "[root] ___ROOT___"
		"[root] cli#lint" -> "[root] ___ROOT___"
		"[root] create-turbo#lint" -> "[root] ___ROOT___"
		"[root] docs#lint" -> "[root] ___ROOT___"
		"[root] eslint-config-turbo#lint" -> "[root] ___ROOT___"
		"[root] eslint-plugin-turbo#lint" -> "[root] ___ROOT___"
		"[root] next-dev-tests#lint" -> "[root] ___ROOT___"
		"[root] tsconfig#lint" -> "[root] ___ROOT___"
		"[root] turbo-ignore#lint" -> "[root] ___ROOT___"
		"[root] turbo-test-utils#lint" -> "[root] ___ROOT___"
		"[root] turbo-types#lint" -> "[root] ___ROOT___"
		"[root] turbo-utils#lint" -> "[root] ___ROOT___"
	}
}
```

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -33,7 +31,7 @@
       "dependsOn": []
     },
     "//#check:fmt": {
-      "dependsOn": ["//#check:prettier", "//#check:toml", "//#lint"]
+      "dependsOn": ["//#check:prettier", "//#check:toml", "lint"]
```
Comment (nathanhammond) [inline turbo.json:34]: We don't even have a `//#check:fmt` script. This is a fake node.

**Review by nathanhammond — APPROVED** [review]
I know it's still in draft, but this seems good to me.

File: turbo.json (hunk 1 above)
Comment (mehulkar) [reply turbo.json:33]: it's invoked here: https://github.com/vercel/turbo/blob/f46865809a783d04cb2df756b44cd0124abbeb84/.github/workflows/test.yml#L860-L861. 

I'm guessing it's for the sole purpose of invoking these dependencies

**Comment by mehulkar** [comment]
This config can't really work without dismantling `check:fmt`. Going to close this down until I have more time to investigate why we have so many `//#` root tasks

**Pull request closed without merging**